src/atc/delta/delta_handle.py

```diff
@@ -0,0 +1,95 @@
+from pyspark.sql import DataFrame
+
+from atc.atc_exceptions import AtcException
+from atc.config_master import TableConfigurator
+from atc.functions import init_dbutils
+from atc.spark import Spark
+
+
+class DeltaHandleException(AtcException):
+    pass
+
+
+class DeltaHandleInvalidName(DeltaHandleException):
+    pass
+
+
+class DeltaHandleInvalidFormat(DeltaHandleException):
+    pass
+
+
+class DeltaHandle:
+    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+        self._name = name
+        self._location = location
+        self._data_format = data_format
+
+        self._validate()
+
+    @classmethod
+    def from_tc(cls, id: str):
+        tc = TableConfigurator()
+        return cls(
+            name=tc.table_name(id),
+            location=tc.table_path(id),
+            data_format=tc.table_property(id, "format", "delta"),
+        )
+
+    def _validate(self):
```

Comment:
"""Validates that the name is either `db`.`table` or just `table`.""" (Maybe do this instead of #-comment)